app/src/main/java/comcmput301f16t01/github/carrier/Requests/DriverViewRequestActivity.java

```diff
@@ -126,16 +125,17 @@ private void setMarkers() {
         map.invalidate();
     }
 
-    // Based on: https://goo.gl/4TKn2y
-    // Retrieved on: November 10th, 2016
-
-    // Updated with: https://goo.gl/h2CKyn
-    // Author: yubaraj poudel
-    // Posted: August 6th, 2016
-    // Retrieved on: November 10th, 2016
 
     /**
      * Asynchronous task to get the route between the two points
+     *
+     * Based on: https://goo.gl/4TKn2y
+     * Retrieved on: November 10th, 2016
+     *
+     * Updated with: https://goo.gl/h2CKyn
+     * Author: yubaraj poudel
```

Comment:
our sources do not need to be in the java docs. Could you move the reuse code to inside the function?